rpc/common/moc_common_nodeinfo.proto

```diff
@@ -27,6 +30,10 @@ message GPU {
 	uint32 count = 3;
 }
 
+message OperatingSystemInfo {
```

Comment:
Just OperatingSystem is good . No need for Info

Reply:
Lets add OSType as a property

Reply:
sku can be uint64

Reply:
https://github.com/microsoft/moc/blob/7bcfbfa2e7ce80a893948f5442734b5b1ba45664/rpc/common/moc_common_computecommon.proto#L29

Reply:
OperatingSystemType is already present as part of NodeInfo.
https://github.com/microsoft/moc/blob/7bcfbfa2e7ce80a893948f5442734b5b1ba45664/rpc/common/moc_common_nodeinfo.proto#L42

Reply:
> Lets add OSType as a property

Added it..

Reply:
> Just OperatingSystem is good . No need for Info

renamed.